src/index.ts

```diff
@@ -221,3 +221,8 @@ export async function isAvailable() {
 export function resetIsAvailableCache() {
   cachedIsAvailableResponse = undefined;
 }
+
+export function requestTimeout(): number {
+  // Increase timeout if we think we're in a Cloud Run or GCF environment:
```

Comment:
Can you over-explain the heck out of this? Link reference issues that could help, too, 'cause you know us... some day we'll go "Oh, there's your problem!" on this and not remember the doom that may be in store.